### PR TITLE
Update savvy to 1.3.0

### PR DESCRIPTION
**Pull request by matuskosut**
Update [`savvy`](https://bioconda.github.io/recipes/savvy/README.html): **1.2.0** &rarr; **1.3.0**
Remove patches intended for 1.2.0 (leave only cmakelists)

[install with bioconda](http://bioconda.github.io/recipes/savvy/README.html) [Conda](https://anaconda.org/bioconda/savvy/files)

Info | Link
-----|-----
Recipe | [`recipes/savvy`](https://github.com//bioconda/bioconda-recipes/tree/bump/savvy/recipes/savvy) (click to view/edit other files)
Releases |[https://github.com/statgen/savvy/tags](https://github.com/statgen/savvy/tags)
Recipe Maintainer(s) | @matuskosut
Author | `@statgen`
***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

**Comment by matuskosut** [comment]
cleanup of patches needed only for 1.2.0 (included in 1.3.0 release by default) should allow easier auto bumps next time,
https://github.com/bioconda/bioconda-recipes/pull/25088

**Comment by matuskosut** [comment]
@BiocondaBot please add label

File: recipes/savvy/meta.yaml [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/statgen/savvy/archive/v{{ version }}.tar.gz
-  sha256: 82faf11665b9150297a4f7d1668df0937a5f6f49f5b8ea0f24f35aa09517c7cf
+  sha256: 5fbf25ff0793de769f12d2a9e0a0f0f40dc3d315cbdbcd16fc6ab90949bfa172
   patches:
     # Fixing postrelease issues
-    - patches/0001-Quick-fix-for-hts_hdr-destruction-bug.patch
```
Comment (bgruening) [inline recipes/savvy/meta.yaml:13]: Oh this looks so nice! :)

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@bioconda-bot please merge

**Comment by bgruening** [comment]
Thanks @matuskosut!

**Pull request merged**